### DISCUSSION TITLE
Rename some Gesture events

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13237] [Ideas]
|Now| Will become|
|:-|:-|
|PointerTouchPadGestureMagnifyEvent|MagnifyEvent|
|PointerTouchPadGestureRotateEvent|RotateEvent
|PointerTouchPadGestureSwipeEvent|SwipeEvent|
|PullGestureEvent|PullEvent
|PullEndsGestureEvent|PullEndsEvent
|ScrollGestureEvent|ScrollEvent
|ScrollGestureEndedEvent|ScrollEndedEvent
|ScrollGestureInertiaStartingEvent|ScrollInertiaStartingEvent

edit after @robloo  suggestion.

**Comment by robloo** [comment]
These changes make seem directionally correct to me but a few comments:

 * Pointer and TouchPad in the same name doesn't make a lot of sense. However, it might be because the gesture is actually triggered by both the pointer and the TouchPad events. That would make sense to me on some platforms but I haven't checked the code.
   * If Magnify/Rotate/Swipe are from both Pointer and TouchPad then remove both so just MagnifyEvent, RotateEvent, etc.
   * If only for TouchPad then your naming above makes sense. However, do we really need to know where it's coming from in this case?
   * It might be better to just have one Magnify/Rotate/Swipe event and then just have the pointer type or kind somewhere in the arguments anyway. So I'm thinking its best to remove TouchPad here too.
 * "Gesture" in the names never made sense. It doesn't follow UWP/WinUI naming convention either: https://learn.microsoft.com/en-us/uwp/api/windows.ui.input.gesturerecognizer.dragging?view=winrt-22621

**Comment by robloo** [comment]
Another thought here. Starting/Ending terminology and usage is completely messed up.

 * WinUI has "ManipulationStarted" and "ManipulationCompleted" along with "ManipulationInertiaStarting" depending on when the event actually fires. "Started" and "Starting" are very important distinctions (see the end below).
 * Start/End are common parings in terminology. System.Range struct, Index, etc. is a good example. However, in this case for gestures Microsoft seems to think "Complete" is a better term. We might want to follow their thinking here.
 * "PullEnds" and "ScrollEnded" is an API bug. Never should different terms like this be used for the same thing. If we keep using start/end pairing both should stay "Ended" however we might want to switch to "Completed" as noted above.
 * As noted elsewhere, we are missing the starting or started pair in several of these events. https://github.com/AvaloniaUI/Avalonia/issues/13208

Also, keep in mind:
 * 'ing' means BEFORE the action has occurred or the event is asynchronous
 * 'ed' means AFTER the action has occurred
